### PR TITLE
Adding busan.hackclub.com / kr.hackclub.com / korea.hackclub.com

### PR DESCRIPTION
**Pull request by juni37**
# Adding busan.hackclub.com / kr.hackclub.com / korea.hackclub.com

## Description

Using the following subdomains:   
busan.hackclub.com -> For Counterspell Busan ONLY.
kr.hackclub.com -> For Hack Club Korea
korea.hackclub.com -> Also for Hack Club Korea.
Currently recruiting members starting from Busan, Korea, for public G10-12 HS students.

### CODE REVIEW

**Comment by alx-alexpark** [comment]
Hey, just checking with counterspell orgs if we grant hackclub.com subdomains to events. 

Also, what page do you plan to put on kr.hackclub.com and korea.hackclub.com? Would that be  your club website?

File: hackclub.com.yaml [hunk 1]
```diff
@@ -1728,11 +1733,18 @@ knit:
 kr:
   ttl: 3600
   type: CNAME
-  value: cname.vercel-dns.com.
```
Comment (alx-alexpark) [inline hackclub.com.yaml:1731]: What was previously on this subdomain?

**Comment by juni37** [comment]
Hi! Yes - it would be Hack Club Korea website that we will put after the event.Sent from my iPhoneOn Oct 9, 2024, at 1:01 PM, Alex Park ***@***.***> wrote:﻿
Hey, just checking with counterspell orgs if we grant hackclub.com subdomains to events.
Also, what site do you plan to put on kr.hackclub.com and korea.hackclub.com? Would that be  your club website?

**Comment by juni37** [comment]
That was previously on the subdomain - we are trying to change it to netlify.Sent from my iPhoneOn Oct 9, 2024, at 1:02 PM, Alex Park ***@***.***> wrote:﻿
@alx-alexpark commented on this pull request.



In hackclub.com.yaml:
> @@ -1728,11 +1733,18 @@ knit:
 kr:
   ttl: 3600
   type: CNAME
-  value: cname.vercel-dns.com.

What was previously on this subdomain?

File: hackclub.com.yaml [hunk 2]
```diff
@@ -726,6 +726,11 @@ boreal:
   type: CNAME
   value: cname.vercel-dns.com.
 
+busan:
+  ttl: 600
+  type: CNAME
+  value: counterspell-busan.netlify.app.
+
```
Comment (alx-alexpark) [inline hackclub.com.yaml:733]: Deven told me that we aren't giving hackclub.com subdomains to Counterspell events. You can always PR into the https://github.com/hackclub/counterspell repo
```suggestion
```

File: hackclub.com.yaml [hunk 3]
```diff
@@ -1728,11 +1733,18 @@ knit:
 kr:
   ttl: 3600
   type: CNAME
-  value: cname.vercel-dns.com.
+  value: counterspell-busan.netlify.app.
```
Comment (alx-alexpark) [inline hackclub.com.yaml:1736]: This is the counterspell Busan site, not the site of your Hack Club in Korea.

File: hackclub.com.yaml [hunk 4]
```diff
@@ -1728,11 +1733,18 @@ knit:
...
+
 ko:
   ttl: 3600
   type: CNAME
   value: cname.vercel-dns.com.
+
+korea:
+  ttl: 3600
+  type: CNAME
+  value: counterspell-busan.netlify.app.
```
Comment (alx-alexpark) [inline hackclub.com.yaml:1746]: This is the counterspell Busan site, not the site of your Hack Club in Korea.

**Review by alx-alexpark — CHANGES_REQUESTED** [review]

File: hackclub.com.yaml (hunk 3 above)
Comment (Muirrum) [reply hackclub.com.yaml:1736]: Also -- ISO 3166 codes should be reserved for hackclub/global

Comment (alx-alexpark) [reply hackclub.com.yaml:1736]: Indeed, are you fine with just the .korea subdomain @juni37?

File: hackclub.com.yaml (hunk 2 above)
Comment (juni37) [reply hackclub.com.yaml:733]: Yup - thanks!

File: hackclub.com.yaml (hunk 3 above)
Comment (juni37) [reply hackclub.com.yaml:1731]: @Muirrum @alx-alexpark It seems like hackclub/global is not maintained at the moment and redirects to the main hackclub.com domain. I am fine with korea.hackclub but could you double check if the repository is currently maintained and enforced?

Comment (alx-alexpark) [reply hackclub.com.yaml:1731]: 2-letter country codes are currently reserved for future use, and yes hackclub/global is still maintained.

Comment (juni37) [reply hackclub.com.yaml:1731]: understood! thanks.

File: hackclub.com.yaml (hunk 1 above)
Comment (Muirrum) [reply hackclub.com.yaml:1731]: Presumably it was reserved for hackclub/global. @juni37 please revert this change

File: hackclub.com.yaml (hunk 4 above)
Comment (Muirrum) [reply hackclub.com.yaml:1741]: Also -- unless it's a hack club for the whole country, this should be a more specific domain to your hack club.

**Review by Muirrum — CHANGES_REQUESTED** [review]

**Comment by juni37** [comment]
Closing this PR as creating new PR with busan.hackclub.com with @Muirrum's suggestion.

**Pull request closed without merging**